src/Builders/Paths/RouteParams/RouteParamsBuilder.php

```diff
@@ -7,11 +7,13 @@
 use GoldSpecDigital\ObjectOrientedOAS\Objects\Parameter;
 use GoldSpecDigital\ObjectOrientedOAS\Objects\Schema;
 use Illuminate\Routing\Route;
+use LaravelJsonApi\Core\Server\Server;
+use LaravelJsonApi\Eloquent\Filters\Concerns\IsSingular;
```

Comment:
Deze use statement is niet nodig.

Reply:
Weggehaald. Wil je php-cs-fixer als dev dependency? Dan kunnen dit soort issues automatisch eruit.